JASP-Engine/JASP/R/ttestbayesianonesample.R

```diff
@@ -605,85 +605,69 @@ TTestBayesianOneSample <- function(dataset=NULL, options, perform="run", callbac
         plottingError[i] <- state$plottingError[index]
         
       } else {
-        
-        result <- try (silent = TRUE, expr = {
-          
-          variableData <- dataset[[ .v(variable) ]]
-          variableData <- variableData[ ! is.na(variableData) ]
-          
-          variableData <- variableData - options$testValue
-          r <- .generalTtestBF(x = variableData, oneSided = oneSided, options = options)
-          bf.raw <- r[["bf"]]
-          tValue[i] <- r[["tValue"]]
-          n[i] <- r[["n1"]]
-          
-          if (bf.type == "BF01")
-            bf.raw <- 1 / bf.raw
-          
-          BF10post[i] <- bf.raw
-          BF <- .clean(bf.raw)
-          
-          if (options$bayesFactorType == "LogBF10") {
-            
-            BF <- log(BF10post[i])
-            BF <- .clean(BF)
-          }
-          
-          error <- .clean(r[["error"]])
-          
-          list(Variable=variable, BF=BF, error=error)
-        })
-        
-        errors <- .hasErrors(dataset, perform, message = "short", type = c('observations', 'variance', 'infinity'),
-                             all.target = variable,
-                             observations.amount = '< 2')
-        
-        if (!identical(errors, FALSE)) {
-            errorMessage <- errors$message
-            status[i] <- "error"
-        } else {
-            errorMessage <- NULL
-        }
-        
-        if(!is.null(errorMessage)){
- 
-          errorFootnotes[i] <- errorMessage
-          
-          index <- .addFootnote(footnotes, errorMessage)
-          
-          result <- list(Variable=variable, BF=.clean(NaN), error="", .footnotes=list(BF=list(index)))
           
-          ttest.rows[[i]] <- result
+          errors <- .hasErrors(dataset, perform, message = "short", type = c('observations', 'variance', 'infinity'),
+                               all.target = variable,
+                               observations.amount = '< 2')
           
-        } else {
-          
-          if (is.na(bf.raw)) {
-            
-            status[i] <- "error"
-            plottingError[i] <- "Plotting is not possible: Bayes factor could not be calculated"
-          } else if(bf.raw == Inf & (options$plotPriorAndPosterior | options$plotBayesFactorRobustness | options$plotSequentialAnalysis | options$plotSequentialAnalysisRobustness)){
-            
-            status[i] <- "error"
-            plottingError[i] <- "Plotting is not possible: Bayes factor is infinite"
```

Comment:
Why did you delete this block?
Infinity in the BF != infinity in the dataset
Right now our messages would not be informative enough.